PROTOTYPE_NOTES.md

```diff
@@ -0,0 +1,130 @@
+# Subgraph Batching Prototype
+Version: Draft 0.1
+
+These notes are designed to maximise the utility of evaluating the subgraph batching prototype both for the customers involved and for Apollo.
+
+Please read these notes carefully before:
+ - Using the prototype router 
+ - Reporting results back to Apollo
+
+The [subgraph batching design document](https://docs.google.com/document/d/1KlGgNCm1sQWc-tYs2oqHauMBIYdrtlpIELySZDAnISE/edit?usp=sharing) may be helpful in evaluating the prototype.
+
+## Using the prototype
+
+### Deployment
+
+#### Accessing the prototype
+
+You can choose to build from source or from one of our pre-built options.
+
+##### Source
+
+The prototype router is publicly available on the main router repo on branch `subgraph-batching-prototype`. It can be accessed and built as normally.
+
+##### Binary
+
+If you want to use a pre-built router, you can access the prototype assets as follows:
+| Artifact | Location |
+| -------- | -------- |
+| Architecture specific binaries | [REPLACE ME] |
+| Docker image | [REPLACE ME] |
+| Helm chart | [REPLACE ME] |
+
+Note: These assets are only available until [REPLACE ME]
+
+### Configuration
+
+Because this is a prototype, we can't update the main `router` documentation and we don't have a way to deliver documentation for prototypes. For this prototype these configuration notes should suffice.
```

Comment:
For future: We can make deploy preview URLs for prototypes, but as you  note the notes here should suffice.

Reply:
Maybe the @apollographql/docs team can help here. We have a docs deploy preview for the main docs repo, so could we have one that does a Router build with the docs from a Router branch

e.g. https://github.com/apollographql/docs/pull/803

Reply:
Yup, our [Netlify settings](https://app.netlify.com/sites/apollo-router-docs/configuration/deploys#branches-and-deploy-contexts) currently only create deploy previews for `dev` and `main`, but we could certainly add other branches to the settings. Would you like one for `garypen/prototype-notes` or `[garypen/2002-subgraph-batching](https://github.com/apollographql/router/tree/garypen/2002-subgraph-batching)`?

Reply:
Could we just do it for all branches?  They have an option for that and it's been handy in the past.

Reply:
I was going to name the branch: `subgraph-batching-prototype`. If we are going to follow some kind of convention, how about `preview-[index]-[feature]`, so this would become `preview-0-subgraph-batching`. If we find major issues that warrant cutting a new preview, then the next preview would become `preview-1-subgraph-batching` etc...

This would take advantage of the make docs available for all branches beginning with `preview` and accommodate multiple revisions of previews for longer lived (or more substantial) preview projects.